Games/types/Mafia/roles/cards/JailTarget.js

```diff
@@ -52,6 +76,9 @@ module.exports = class JailTarget extends Card {
         leader: true,
         priority: MEETING_PRIORITY_JAIL,
         shouldMeet: function () {
+          if (this.player.hasItem("Handcuffs")) {
```

Comment:
this is not the right place to implement this check, we'd have to duplicate the logic for banquet seance

#256